src/main/webapp/app/entities/course/course.component.ts

```diff
@@ -41,6 +44,10 @@ export class CourseComponent implements OnInit, OnDestroy {
         this.eventManager.destroy(this.eventSubscriber);
     }
 
+    get today() {
```

Comment:
This works, but I would prefer it if we use the ngx-moment implementation: https://www.npmjs.com/package/ngx-moment#amisbefore-and-amisafter-pipe
The advantage is we can do all of it in the template and don't need to implement it in the component.

Reply:
I now use the ngx-moment implementation as suggested